engine/src/main/resources/application.properties

```diff
@@ -73,9 +73,9 @@ quarkus.rest-client.rbac-s2s.read-timeout=120000
 
 # IT User service
 # these entries are needed because of a bug in quarkus: https://github.com/quarkusio/quarkus/issues/8384
-quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".url=${QUARKUS_REST_CLIENT_IT_S2S_URL:FILL_ME}/v2
-quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE:FILL_ME}
-quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD:FILL_ME}
+%prod.quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".url=${QUARKUS_REST_CLIENT_IT_S2S_URL:FILL_ME}/v2
+%prod.quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE:FILL_ME}
+%prod.quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD:FILL_ME}
```

Comment:
These config values were causing issues during tests so they will only be active with the `prod` profile from now on.